docs/contribute/unit-tests.md

```diff
@@ -1,23 +1,81 @@
+## Dependencies
```

Comment:
The document could use a title, e.g. `# Running unit tests`. 

terminology nit: Are these really unit tests? I know, the definition depends on the context, but in a (mostly) rust project, I'd use the term ["unit test" for other types of tests](https://doc.rust-lang.org/book/ch11-03-test-organization.html#unit-tests). Maybe we just call them tests? 🤷

Reply:
The document could also use a short opening paragraph. Something along the lines of ...

This guide covers 
- building the lind toolchain, 
- running the test suite,
- interpreting its results,
- and adding new tests. 

Lind is currently only available for the AMD64 platform. To run tests on any host, we use Docker.